riscv/processor.cc

```diff
@@ -55,7 +55,10 @@ processor_t::processor_t(const isa_parser_t *isa, const cfg_t *cfg,
   }
 #endif
 
-  parse_varch_string(cfg->varch);
+  VU.VLEN = isa->get_vlen();
+  VU.ELEN = isa->get_elen();
+  VU.vlenb = isa->get_vlen() / 8;
+  VU.vstart_alu = 1;
```

Comment:
Thinking about this some more, I'm questioning whether this is the right choice to pick.  There are cases where supporting vstart>0 adds complexity, and it's safe to say that there will be more over time.  So, why pick the option that makes Spike harder to maintain?

Reply:
Updated to set `start_alu=0`